### PR TITLE
Updating the readme to reflect the Atlas Device SDK branding

### PR DESCRIPTION
**Pull request by kraenhansen**
## What, How & Why?

This updates our readme to reflect the Atlas Device SDK branding: https://github.com/realm/realm-js/blob/b2dd0b684a8601d968bc8d8f634364383b159cb2/packages/realm/README.md

### CODE REVIEW

File: packages/realm/README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-</picture>
+> [!NOTE]
+> Realm is now Atlas Device SDK - [Learn More](https://www.mongodb.com/blog/post/realm-now-part-atlas-platform?utm_medium=github_atlas_CTA&utm_source=realm_js_github)
 
-# About Realm Database
+<img align="right" style="min-width:150px;width:30%;" src="https://raw.githubusercontent.com/realm/realm-js/main/media/atlas-device-sync.svg" alt="Atlas Device Sync">
 
-Realm is a mobile database that runs directly inside phones, tablets or wearables.
-This project hosts the JavaScript versions of [Realm](https://www.mongodb.com/docs/realm/). Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+# Atlas Device SDK
```
Comment (kraenhansen) [inline packages/realm/README.md:6]: I was contemplating putting "(for JavaScript & TypeScript)" here as well, but it felt a bit verbose in a title.

**Review by kneth — APPROVED** [review]

File: packages/realm/README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+
+Atlas Device SDK is a suite of app development tools optimized for data access and persistence on mobile and edge devices. Use the SDKs to build data-driven mobile, edge, web, desktop, and IoT apps.
```
Comment (krollins-mdb) [inline packages/realm/README.md:8]: What docs has been told and what we're working with is that Realm is the device data persistence layer part of Atlas Device SDK. Check out the Introduction page in the docs: https://www.mongodb.com/docs/realm/introduction/

File: packages/realm/README.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
+
+This project hosts the JavaScript / TypeScript versions of the SDK. Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+
+An integral part of this SDK is a mobile database that runs directly inside phones, tablets or wearables. This used to be called Realm and is still referred to as Realm throughout our codebase as we're currently focused on improving the experience over renaming everything, causing lots of breaking changes, etc.
```
[the remark this answers is not in the document]
Comment (kraenhansen) [reply packages/realm/README.md:12]: Okay. That makes sense. But then I definitely think our announcing post should be clarified to reflect that nuance: https://www.mongodb.com/blog/post/realm-now-part-atlas-platform

[the remark this answers is not in the document]
Comment (kraenhansen) [reply packages/realm/README.md:12]: It's basically not Realm (the database) but the client for what was formerly know as Stitch which has a new name then.

**Comment by kraenhansen** [comment]
I've drafted this for now as I want to rephrase with the clarification that @krollins-mdb commented 👍

**Comment by kraenhansen** [comment]
I've tried to clarify the relationship between Atlas Device SDKs and the Realm DB, please give it another look 🤞

File: packages/realm/README.md [hunk 4]
```diff
@@ -30,24 +40,24 @@ If you want to contribute to any of our packages, you are welcome to do so. We w
...
+* Any other release of [`realm`](https://www.npmjs.com/package/realm) not covered by tier 1
 
 ### Tier 3 - experimental
 
 The third tier covers our experimental packages. We work on them occasionally, and they are likely to change radically when we do.
 
-* [Realm Babel plugin](https://www.npmjs.com/package/@realm/babel-plugin)
-* [Realm Flipper plugins](https://www.npmjs.com/package/realm-flipper-plugin)
+* [`@realm/babel-plugin`](https://www.npmjs.com/package/@realm/babel-plugin)
+* [`realm-flipper-plugin`](https://www.npmjs.com/package/realm-flipper-plugin)
```
Comment (kneth) [inline packages/realm/README.md:60]: I think we should remove it

**Review by kneth — APPROVED** [review]

File: packages/realm/README.md [hunk 5]
```diff
@@ -1,24 +1,34 @@
...
+
 <picture>
-    <source srcset="https://github.com/realm/realm-js/blob/HEAD/media/logo-dark.svg" media="(prefers-color-scheme: dark)" alt="realm by MongoDB">
-    <img src="https://github.com/realm/realm-js/blob/HEAD/media/logo.svg" alt="realm by MongoDB">
+    <source srcset="https://raw.githubusercontent.com/realm/realm-js/main/media/logo-dark.svg" media="(prefers-color-scheme: dark)" alt="realm by MongoDB">
+    <img src="https://raw.githubusercontent.com/realm/realm-js/main/media/logo.svg" alt="realm by MongoDB">
 </picture>
 
-# About Realm Database
+# The Realm database
```
Comment (elle-j) [inline packages/realm/README.md:9]:
```suggestion
# The Realm Database
```

File: packages/realm/README.md [hunk 6]
```diff
@@ -1,24 +1,34 @@
...
+
+Realm is a mobile database that runs directly inside phones, tablets or wearables. This project hosts the JavaScript & TypeScript implementation of Realm. Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+
+## What is the Atlas Device SDKs?
```
Comment (elle-j) [inline packages/realm/README.md:13]:
```suggestion
## What are the Atlas Device SDKs?
```

File: packages/realm/README.md [hunk 7]
```diff
@@ -1,24 +1,34 @@
...
+
+<img align="right" style="min-width:150px;width:20%;" src="https://raw.githubusercontent.com/realm/realm-js/main/media/atlas-device-sync.svg" alt="Atlas Device Sync">
 
-Realm is a mobile database that runs directly inside phones, tablets or wearables.
-This project hosts the JavaScript versions of [Realm](https://www.mongodb.com/docs/realm/). Currently, we support React Native (JSC & Hermes on iOS & Android), Node.js and Electron (on Windows, MacOS and Linux).
+The [Atlas Device SDKs](https://www.mongodb.com/docs/realm/sdk/) are a connection of language & platform specific SDKs, each with a suite of app development tools optimized for data access and persistence on mobile and edge devices. Use the SDKs to build data-driven mobile, edge, web, desktop, and IoT apps.
```
Comment (elle-j) [inline packages/realm/README.md:17]:
```suggestion
The [Atlas Device SDKs](https://www.mongodb.com/docs/realm/sdk/) are a connection of language and platform specific SDKs, each with a suite of app development tools optimized for data access and persistence on mobile and edge devices. Use the SDKs to build data-driven mobile, edge, web, desktop, and IoT apps.
```

File: packages/realm/README.md [hunk 8]
```diff
@@ -30,24 +40,24 @@ If you want to contribute to any of our packages, you are welcome to do so. We w
...
 ### Tier 3 - experimental
 
 The third tier covers our experimental packages. We work on them occasionally, and they are likely to change radically when we do.
 
-* [Realm Babel plugin](https://www.npmjs.com/package/@realm/babel-plugin)
-* [Realm Flipper plugins](https://www.npmjs.com/package/realm-flipper-plugin)
+* [`@realm/babel-plugin`](https://www.npmjs.com/package/@realm/babel-plugin)
+* [`realm-flipper-plugin`](https://www.npmjs.com/package/realm-flipper-plugin)
 
 ## Documentation
```
Comment (elle-j) [inline packages/realm/README.md:62]: In the first paragraph here it still says "Realm React Native SDK" and "Realm Node.js SDK". (Couldn't comment on the exact lines.)

File: packages/realm/README.md [hunk 9]
```diff
@@ -83,9 +93,7 @@ class Task extends Realm.Object<Task, "description"> {
 
 ### Integration with React Native
 
-Realm is a general SDK which provide you persistence of objects and the capability to perform advanced queries on the objects. You can have tighter integration with React Native by using [@realm/react](https://www.npmjs.com/package/@realm/react).
-
-Moreover, we have a [Flipper plugin](https://www.npmjs.com/package/realm-flipper-plugin) to help you inspect, query and modify your Realm files while debugging your app on a simulator or a physical device. The plugin is still in an early stage so expect rough edges.
+Realm is a general SDK which provide you persistence of objects and the capability to perform advanced queries on the objects. You can have tighter integration with React Native by using [`@realm/react`](https://www.npmjs.com/package/@realm/react).
```
Comment (elle-j) [inline packages/realm/README.md:96]:
```suggestion
Realm is a general SDK which provides persistence of objects and the capability to perform advanced queries on the objects. You can have easier integration with React Native by using [`@realm/react`](https://www.npmjs.com/package/@realm/react).
```

**Review by elle-j — APPROVED** [review]
👌

File: packages/realm/README.md (hunk 8 above)
Comment (krollins-mdb) [reply packages/realm/README.md:62]: And the heading for this section has the old names too.
"Atlas Device SDK for React Native and Atlas Device SDK for Node.js" as a heading is a bit much though. We could probably simplify to "Atlas Device SDKs for React Native and Node.js".

File: packages/realm/README.md (hunk 9 above)
Comment (krollins-mdb) [inline packages/realm/README.md:96]:
```suggestion
The Atlas Device SDK for React Native provides you with object persistence and
advanced queries for persisted objects. You can have tighter integration with React Native by using [`@realm/react`](https://www.npmjs.com/package/@realm/react).
```

**Review by krollins-mdb — APPROVED** [review]
Thanks for taking the time to think about how to present the new branding to users, @kraenhansen! I only have a couple of minor suggestions.

File: packages/realm/README.md (hunk 4 above)
Comment (kraenhansen) [reply packages/realm/README.md:60]:
```suggestion
```

Comment (kraenhansen) [reply packages/realm/README.md:60]: I'll leave actually removing it from the repository, to a separate PR 🙂

File: packages/realm/README.md (hunk 8 above)
Comment (kraenhansen) [reply packages/realm/README.md:61]: Updated 👍

**Pull request merged**